app/models/forms/BankDetailsForm.scala

```diff
@@ -83,12 +84,44 @@ object BankDetailsForm {
   ))
 
   // BARs FormErrors
-  val accountNumberNotWellFormatted: FormError = FormError("bars", "validate.accountNumberIsWellFormatted.no")
-  val sortCodeNotPresentOnEiscd: FormError = FormError("bars", "validate.sortCodeIsPresentOnEISCD.no")
-  val sortCodeDoesNotSupportsDirectDebit: FormError = FormError("bars", "validate.sortCodeSupportsDirectDebit.no")
-  val nameDoesNotMatch: FormError = FormError("bars", "verify.nameMatches.no")
-  val accountDoesNotExist: FormError = FormError("bars", "verify.accountExists.no")
-  val sortCodeOnDenyList: FormError = FormError("bars", "verify.sortCodeOnDenyList")
-  val otherBarsError: FormError = FormError("bars", "verify.otherError")
+  val barsNameOverride: FormError = FormError("name", "bars.confirm")
+  val barsSortCodeOverride: FormError = FormError("sortCode", "bars.confirm")
+  val barsAccountNumberOverride: FormError = FormError("accountNumber", "bars.confirm")
+
+  val accountNumberNotWellFormatted: FormErrorWithFieldMessageOverrides =
```

Comment:
do we need a concept of a `FormErrorWithFieldMessageOverrides`? The original `Form` class from play already lets you define more than one form error in it doesn't it?

Reply:
Probably easier to talk through it.